swagger.yaml

```diff
@@ -3643,7 +3643,11 @@ paths:
           name: platform_type
           description: The provider platform type.
           type: string
-          enum: [ 'baremetal', 'none', 'nutanix', 'vsphere', 'oci', 'external' ]
+          enum: [ 'baremetal', 'none', 'nutanix', 'vsphere', 'external' ]
+        - in: query
+          name: external_platform_name
+          description: External platform name when platform type is set to external. The value of this parameter will be ignored if platform_type is not external.
+          type: string
```

Comment:
@avishayt @ammont82 I added this parameter to get the features linked to an external integration (e.g.: oci). It looks like this API is already pretty loose, so I won't throw an error if this field is set and platformType is different fron external. Let me know if it looks ok

Reply:
@adriengentil then we need to send this parameter when platform type='external' and add 'external_platform_name='oci'' for the OCI case?

Reply:
As discussed, yes!